src/Api/Search.php

```diff
@@ -456,6 +456,54 @@ public static function getLocationsFilter()
             return array();
         }
 
+        public static function getAccountAssosiations($search)
+        {
+            $activate = Oauth2\Activate::instance();
+            $apiParams = $activate::$params;
+
+            $accessToken = $activate->getAuthorizeToken()['token'];
+            $appId = Settings::instance()->getSettingsOption('account', 'app_id');
+            $instance = Settings::instance()->getSettingsOption('account', 'instance');
+
+            $apiParams['accessToken'] = $accessToken;
+            $apiParams['clientId'] = $appId;
+            $apiParams['instance'] = $instance;
+
+            $authorizationHeaders = SDKClient::setHeaders($apiParams);
+            $client = new SDKClient($apiParams['apiUri'], $authorizationHeaders);
+
+            $gql = 'query partnerAccounts {
+              accounts(filters: [
+                {field: name, operation: like, value: "%' . $search . '%"}
```

Comment:
we can use variables instead of string interpolation https://graphql.org/graphql-js/passing-arguments/

I think we talked about this in the past though